polymer/foam-select.html

```diff
@@ -0,0 +1,138 @@
+<link rel="import" href="./bower_components/polymer/polymer.html">
+
+<dom-module id="foam-select">
```

Comment:
Would be good to have documentation about what this element does as foam-select is not a very descriptive name.